dolphinscheduler-alert/pom.xml

```diff
@@ -31,14 +31,12 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
     <dependencies>
+        <!-- dolphinscheduler -->
         <dependency>
             <groupId>org.apache.dolphinscheduler</groupId>
-            <artifactId>dolphinscheduler-remote</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.dolphinscheduler</groupId>
-            <artifactId>dolphinscheduler-common</artifactId>
+            <artifactId>dolphinscheduler-service</artifactId>
```

Comment:
Hi, ZhuangChong, if we add the dependency of dolphinscheduler-service in the alert, it seems that zookeeper, quartz-jobs,etc. will also be added to the alert module. I am not sure if this is what you need.